projects.md

```diff
@@ -1,3 +1,27 @@
+---
+layout: contribute
+title: Mentoring Projects
+---
+
+## Mentoring Projects
+
+This page details potential projects and their mentors, for participants of
+internship programs such as [Google Summer of Code](https://www.google-melange.com/),
+[Outreachy](https://www.gnome.org/outreachy/), and so forth. Each program will
+have it's own requirements and application process - this page simply lists the
+currently available projects.
+
+## I'm interested - how can I get familiar with Foreman?
+
+* Grab a cup of your favorite beverage
+* Skim through the [manual](/documentation.html)
+* Install Foreman on a virtual machine or your own computer
+* In the meantime, watch a few [demo videos](/media.html)
+* If the installation hasn't finished yet, read a couple posts from the [mailing list](/support.html#Mailinglists)
+* Join us on [IRC](/support.html#IRC) at #theforeman and #theforeman-dev on Freenode. Don't be shy; tell us you're interested in one of our open mentoring projects.
```

Comment:
i think i'll split it into to lines - the two channels have different purposes after all. good catch.